modules/clientValidation.js

```diff
@@ -8,6 +8,7 @@ const requiredFields = [
   { input: email, message: 'Email is required' },
```

Comment:
- [ ] Good job making the email a required field. However, I can submit a form with my email in uppercase. Please add a validation so that if a user types in an email address in uppercase, an error would be displayed.

Reply:
Here is the regex that allows only lowercase on line 30 
  const re = /^[a-z0-9.!#$%&'*+=?^_`{|}~-]+@[a-z0-9-]+(?:\.[a-z90-]+)*$/;